<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="statusTip">
   <string>欢迎</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCommandLinkButton" name="searchButton">
        <property name="statusTip">
         <string>新增数据库中图书</string>
        </property>
        <property name="text">
         <string>查找图书</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/resource/add.png</normaloff>:/resource/add.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="borrowButton">
        <property name="statusTip">
         <string>处理借书事务</string>
        </property>
        <property name="text">
         <string>借书事务</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/resource/borrow.png</normaloff>:/resource/borrow.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="returnButton">
        <property name="statusTip">
         <string>处理还书事务</string>
        </property>
        <property name="text">
         <string>还书事务</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/resource/return.png</normaloff>:/resource/return.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="updateButton">
        <property name="statusTip">
         <string>新增数据库中图书</string>
        </property>
        <property name="text">
         <string>刷新</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/resource/renew1.png</normaloff>:/resource/renew1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="quitButton">
        <property name="statusTip">
         <string>退出程序</string>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/resource/quit.png</normaloff>:/resource/quit.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>774</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
